src/V3EmitMk.cpp

```diff
@@ -60,13 +64,19 @@ class EmitMk {
         of.puts("VM_THREADS = ");
         of.puts(cvtToStr(v3Global.opt.threads()));
         of.puts("\n");
-        of.puts("# Tracing output mode?  0/1 (from --trace)\n");
+        of.puts("# Tracing output mode?  0/1 (from --trace/--trace-fst)\n");
         of.puts("VM_TRACE = ");
         of.puts(v3Global.opt.trace() ? "1" : "0");
         of.puts("\n");
-        of.puts("# Tracing threaded output mode?  0/1 (from --trace-fst-thread)\n");
-        of.puts("VM_TRACE_THREADED = ");
-        of.puts(v3Global.opt.traceFormat().threaded() ? "1" : "0");
+        of.puts("# Tracing threaded output mode?  0/1/N threads (from --trace-thread)\n");
+        of.puts("VM_TRACE_THREADS = ");
+        of.puts(!v3Global.opt.traceThreads()
+                    ? "0"
+                    : cvtToStr(v3Global.opt.traceThreads() - v3Global.opt.traceFormat().fst()));
+        of.puts("\n");
+        of.puts("# Separate FST writer thread? 0/1 (from --trace-fst with --trace-thread > 0)\n");
+        of.puts("VM_TRACE_FST_WRITER_THREAD = ");
+        of.puts(v3Global.opt.traceThreads() && v3Global.opt.traceFormat().fst() ? "1" : "0");
```

Comment:
Perhaps cleaner to have a VM_TRACE_FST that is just format fst, and then look for both being set?

Reply:
Actually, with `--trace-fst --trace-threads 1`, only VM_TRACE_FST_WRITER_THREAD is set, as that gives optimal performance for only 1 extra thread.